src/users/interfaces/user.interface.ts

```diff
@@ -0,0 +1,34 @@
+export type User = ManagedUser | UnmanangedUser;
+
+interface ManagedUser extends BaseUser {
+  user_type: 'managed';
+  identities: [Identity];
+  sso_profile_id: string;
+}
+
+interface UnmanangedUser extends BaseUser {
+  user_type: 'unmanaged';
+  identities: Identity[];
+}
+
+interface BaseUser {
+  object: 'user';
+  id: string;
+  email: string;
+  first_name?: string;
+  last_name?: string;
+  email_verified_at?: string;
```

Comment:
This is only a property on the `UnmanagedUser`.

Reply:
Ohh, still an optional field on the UnmanagedUser i assume?

Reply:
Yep!